src/styles/global.css

```diff
@@ -2,6 +2,11 @@
 @tailwind components;
 @tailwind utilities;
 
+:root {
```

Comment:
Are these colors consistent with the brand guidelines? If you need a link, it should be in the #website channel on discord

Reply:
The colors themselves were, but the naming wasn't since they were hardcoded previously and I didn't see the branding guidelines. I've read the sections concerning color since, and have renamed the variables appropiately both in CSS and Tailwind, see https://github.com/LadybirdBrowser/ladybird.org/pull/115/commits/9d0589a9210dddbc42a905432c2bdda51ec40dba.